app/services/db-export/schema-export.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+/**
+ * Exports an entire db schema and cleans up the files after upload
+ * @module SchemaExportService
+ */
+
+const path = require('path')
+const os = require('os')
+
+const DeleteFolderService = require('./delete-folder.service.js')
+const FetchTableNames = require('../db-export/fetch-table-names.service.js')
+const SendToS3BucketService = require('../db-export/send-to-s3-bucket.service.js')
+const ExportCompressedTableService = require('./export-compressed-table.service.js')
```

Comment:
Just a small tweak to alphabetise the requires.
```suggestion
const os = require('os')
const path = require('path')

const DeleteFolderService = require('./delete-folder.service.js')
const ExportCompressedTableService = require('./export-compressed-table.service.js')
const FetchTableNames = require('../db-export/fetch-table-names.service.js')
const SendToS3BucketService = require('../db-export/send-to-s3-bucket.service.js')
```